lib/foreman_maintain/utils/command_runner.rb

```diff
@@ -59,12 +59,15 @@ def execution_error
 
       private
 
+      # rubocop:disable Metrics/MethodLength
       def run_interactively
         # use tmp files to capture output and exit status of the command when
         # running interactively
         log_file = Tempfile.open('captured-output')
         exit_file = Tempfile.open('captured-exit-code')
-        Kernel.system("script -qc '#{full_command}; echo $? > #{exit_file.path}' #{log_file.path}")
+        Kernel.system(
+          "bash -c '#{full_command}; echo $? > #{exit_file.path}' | tee -i #{log_file.path}"
```

Comment:
@jjeffers I am unable to find the difference between these commands apart from the piped `tee`. Because `script -c` runs the command instead of interactive shell and the `bash -c` is also doing the same. The `script` records the logs and exit status in given files. Bit more explanation of command change should help, thank you!

Reply:
@upadhyeammit hey I made this PR based on feedback from a user. I am not sure about why they changed the command from `script` to `bash`, etc. I didn't notice that before. I was looking at the `tee` near the end.

Let me see if I can figure out if the change is meaningful. Thanks!

Reply:
@upadhyeammit Testing with `bash` showed the change was not helpful. So I restored `script -qc`.